Formula/hyperscan.rb

```diff
@@ -22,6 +22,9 @@ class Hyperscan < Formula
   def install
     mkdir "build" do
       args = std_cmake_args
+      args += %w[
+        -DBUILD_STATIC_AND_SHARED=on
+      ]
```

Comment:
To write the following is faster in Ruby.
```
args << "-DBUILD_STATIC_AND_SHARED=on"
```

Reply:
yup..was just trying to follow the pattern below: https://github.com/Homebrew/homebrew-core/pull/20054/files#diff-c6d2052c00252951884d401ab75abdceR35

Reply:
That is slower than the following. And following is more compact.
```
args << "-DCMAKE_BUILD_TYPE=Debug" << "-DDEBUG_OUTPUT=on"
```

Reply:
Would you like me to update it? I was following the existing pattern since I didn't commit the original formula.

Reply:
I think original part is not your task.

Reply:
@s172262 is correct with the first comment. When it's just a single argument don't use an array. If there are multiple arguments that cause the line to dip over 80 characters, use an array.